torch/_torch_docs.py

```diff
@@ -745,6 +745,23 @@ def merge_dicts(*dicts):
     tensor([0.0000, 0.0000, 0.0000, 1.0000, 1.0000, 0.0000, 0.5000])
 """)
 
+add_docstr(torch.bitwise_not,
+           r"""
+bitwise_not(input, out=None) -> Tensor
+
+Computes the bitwise NOT of the given input tensor. The input must be of
+integer or Boolean types.
```

Comment:
Maybe `integral` is a better word than `integer`? `bitwise_not` works on `torch.int` and `torch.long`, right?